README.md

```diff
@@ -13,23 +13,32 @@ Note that you MUST [`source`](https://linuxize.com/post/bash-source-command/) (o
 
 Meaning, *the period at the beginning of those lines above is a critical, vital part of using this application*.
 
+If using Fish, you will need to use a slight workaround due to how Fish's [source](https://fishshell.com/docs/current/cmds/source.html) command only resolves relative paths from the current directory:
+
+    . (which aws-cli-mfa) $AWS_MFA_ARN 123456
+
 An easy way to test that everything is working is simply:
 
     aws s3 ls
 
 Note that aws cli commands will only work in the terminal you `source`-d this script in. If you want to use `aws` in other terminals, you will need to pass `--profile mfa` at the end ala `aws s3 ls --profile mfa` or you can `export AWS_PROFILE=mfa` in the appropriate profile/rc file to make that your default aws profile (for everything!).
 
+
```

Comment:
Is there a reason for this extra newline?

Reply:
Nope, fixed.

Reply:
Are you sure it's fixed? Still showing up

Reply:
Rats, not sure how that snuck back in.  Now it is actually fixed :D